### PR TITLE
add simple test suite for BE

### PR DESCRIPTION
**Pull request by tanujtiwari98**
Added test suite for BE

### CODE REVIEW

File: api/app/api/utils/sql_gen/tests/test_txt_to_sql.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    (
+        "10 highest crime cities in California",
+        [lambda result: any(sub_result.get('city', '') == 'Los Angeles' for sub_result in result),
+         lambda result: any(sub_result.get('city', '') != 'Los Gatos' for sub_result in result)]
+    )
+]
+
+
+@pytest.mark.parametrize("input_query, expected_output_evaluate_funcs", inputs)
+def test_always_fails(input_query, expected_output_evaluate_funcs):
```
Comment (tanujtiwari98) [inline api/app/api/utils/sql_gen/tests/test_txt_to_sql.py:24]: this might be going a little overboard and being too general but lmk what y'all think

Comment (alex-sun-kuo) [reply api/app/api/utils/sql_gen/tests/test_txt_to_sql.py:24]: hahaha the los gatos check. looks ok to me

File: api/requirements.txt
Comment (alex-sun-kuo) [inline api/requirements.txt:1]: @tanujtiwari98 can you add a `pytest` version in `requirements.txt` like the other packages?

Comment (tanujtiwari98) [reply api/requirements.txt:1]: Done

**Pull request merged**